dist/index.js

```diff
@@ -65937,4 +65938,4 @@ if (__filename.endsWith('index.js')) { run() }
 
 module.exports = __webpack_exports__;
 /******/ })()
-;
\ No newline at end of file
+;
```

Comment:
There is an extra newline here which will most likely cause CI to fail.

Anyway I'll use https://github.com/ruby/setup-ruby/pull/596 since that was created earlier and both PRs are basically the same.
Thank you for the PR though.